packages/api-server/src/convert-tx.ts

```diff
@@ -35,6 +44,35 @@ export interface PolyjuiceTransaction {
   s: HexString;
 }
 
+export function polyjuiceRawTransactionToApiTransaction(
+  rawTx: HexString,
+  ethTxHash: Hash,
+  tipBlockHash: Hash,
+  tipBlockNumber: bigint,
+  fromEthAddress: HexString
+): EthTransaction {
+  const tx: PolyjuiceTransaction = decodeRawTransactionData(rawTx);
+
+  const pendingBlockHash = bumpHash(tipBlockHash);
+  const pendingBlockNumber = new Uint64(tipBlockNumber + 1n).toHex();
+  return {
+    hash: ethTxHash,
+    blockHash: pendingBlockHash,
+    blockNumber: pendingBlockNumber,
+    transactionIndex: "0x0",
+    from: fromEthAddress,
+    to: tx.to == "0x" ? null : tx.to,
+    gas: tx.gasLimit === "0x" ? "0x0" : "0x" + BigInt(tx.gasLimit).toString(16),
+    gasPrice: tx.gasPrice === "0x" ? "0x0" : tx.gasPrice,
+    input: tx.data,
+    nonce: tx.nonce === "0x" ? "0x0" : tx.nonce,
+    value: tx.value === "0x" ? "0x0" : tx.value,
+    v: +tx.v % 2 === 0 ? "0x1" : "0x0",
```

Comment:
Why do we have to give them the default values? What if I just pass the original tx parameters like below?:

```ts
    to: tx.to,
    gas: tx.gasLimit,
    gasPrice: tx.gasPrice,

    nonce: tx.nonce,
    value: tx.value,
    v: tx.v,
```

Reply:
If `gasLimit` is `0x`, should return `0x0`, `0x` is a wrong format